doc/rst/developer/bestPractices/EditionsDev.rst

```diff
@@ -18,8 +18,111 @@ When creating new editions, be sure to:
   - Update :ref:`edition-changes` to list those changes under this new release
     name instead of under :ref:`preview-changes`.
 
-- When https://github.com/chapel-lang/chapel/issues/27336 gets implemented, be
-  sure to update it as well.
+- When `--flag-edition-changes <TransitionFlag>`_ gets implemented, be sure to
+  update it as well.
+
+-----------------------
+What Goes in an Edition
+-----------------------
+
+When making a change to the language and/or standard library, its important
+to consider how that change fits into an edition. Editions define the state of
+the language at a point in time, so breaking changes to stable code should be
+associated with an edition.
+
+It is important to define what constitutes a "breaking change" and what belongs
+in an edition versus what can just be changed or added.
+We consider a feature to be stable if it is not marked unstable.
+
+* **Breaking Changes**
+
+   * altering the behavior of a stable feature
+
+   * deprecation or removal of a stable feature
+
+* **Non-Breaking Changes**
+
+   * altering the behavior of an unstable feature
+
+   * deprecation or removal of an unstable feature
+
+   * addition of a new feature
+
+.. note::
+
+   On a case by case basis, some changes that would normally be considered
+   non-breaking may be treated as breaking changes. For example, if an
+   unstable feature is widely used in practice, changing its behavior may be
+   considered a breaking change.
+
+The following provides a set of for several scenarios of language evolution. In
+this, we do not distinguish between language features and standard library
+features.
+
+* **Changing the behavior of a stable feature** (`Breaking Change`)
+
+    A stable feature should only have its behavior change in the ``preview``
+    edition. The ``preview`` edition collects changes that will be included in
+    the next edition.
+
+    .. note::
+
+        An unimplemented feature is the
+        `--flag-edition-changes <TransitionFlag>`_ flag, which will warn users
+        in a given edition about code that will need to be migrated in the next
+        edition.
```

Comment:
Similar comment here about using a name that hasn't been decided on.  It's less important here than it was earlier, though, since the point of this note is centered around the flag in question.  Using a name we don't ultimately end up choosing will only potentially have the downside of not recognizing when to treat the note as resolved